api/iscsi.proto

```diff
@@ -0,0 +1,349 @@
+syntax = "proto3";
+
+package api;
+
+import "errors.proto";
+
+service IscsiCSIProxyService {
```

Comment:
I see both `Iscsi` and `IScsi` below. Can you make the capitalization consistent?